### PR TITLE
Fix translations

### PR DESCRIPTION
**Pull request by Ayub-Khan**
-removed i18n generate merge options

-updated header

-updated-translations

### CODE REVIEW

File: ecommerce/conf/locale/config.yaml [hunk 1]
```diff
@@ -94,9 +94,3 @@ ignore_dirs:
     - node_modules
     - static/bower_components
     - static/build
-
-generate_merge:
```
Comment (amangano-edx) [inline ecommerce/conf/locale/config.yaml:98]: Why does this need to be removed?

**Review by amangano-edx — APPROVED** [review]
This seems ok. Do you plan on adding the make target for `i18n_tool validate` in a later commit?

**Comment by Ayub-Khan** [comment]
here is the pr for validate command https://github.com/edx/ecommerce/pull/1668

File: ecommerce/conf/locale/config.yaml (hunk 1 above)
Comment (Ayub-Khan) [reply ecommerce/conf/locale/config.yaml:98]: As it is not used and causes problems for i18n-tool validate command

**Pull request merged**